tests/internal/test_cache.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from meatie.internal.cache import Cache
+
+
+class TestCache:
```

Comment:
Would you mind moving tests outside the class? We are not using here any state or setup/teardown. Moving tests outside the calss with simplify the file.